src/extension-runners/chromium.js

```diff
@@ -217,8 +242,10 @@ export class ChromiumExtensionRunner {
         if (msg.type === 'webExtReloadAllExtensions') {
           const devExtensions = await getAllDevExtensions();
           for (var ext of devExtensions) {
+            // just removing await .. fix and keep feature?
```

Comment:
The `await` is necessary. Otherwise there is no guarantee that all extensions have been reloaded.

Assuming that `reloadExtension` never rejects, the logic would be

```
await Promise.all(devExtensions.map(ext => reloadExtension(ext.id)));
```